Dockerfile.slim

```diff
@@ -11,7 +11,7 @@ RUN apk --no-cache add --update bash wget unzip python=2.7.12-r0 && \
   CLOUDSDK_PYTHON="python2.7" /opt/google-cloud-sdk/install.sh --usage-reporting=false --bash-completion=false --disable-installation-options && \
   rm -rf ~/.config/gcloud
 
-RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl && \
+RUN wget https://storage.googleapis.com/kubernetes-release/release/stable.txt && wget https://storage.googleapis.com/kubernetes-release/release/$(cat stable.txt)/bin/linux/amd64/kubectl && \
```

Comment:
Should we run `rm stable.txt` as another RUN step?

Reply:
yup. good call